spacq/devices/keithley/dm2100.py

```diff
@@ -0,0 +1,119 @@
+import logging
+log = logging.getLogger(__name__)
+
+from spacq.interface.resources import Resource
+from spacq.tool.box import Synchronized
+
+from ..abstract_device import AbstractDevice
+from ..tools import quantity_wrapped
+
+"""
+Keithley 2100 Digital Multimeter
+
+Obtain measurements from the multimeter.
+"""
+
+
+class DM2100(AbstractDevice):
+	"""
+	Interface for Keithley 2100 DM.
+
+	Note: Currently supports only DC voltage readings.
+	"""
+
+	allowed_nplc = set([0.02, 0.1, 1.0, 10.0])
```

Comment:
What does ```nplc``` stand for, can we use the full name of the variable, or put the definition of the shortcut in the class docstring?